### ISSUE TITLE
License

### ISSUE THREAD

**Issue by cmorty** [DVDPT/libMPSSE-.Net-Wrapper#1]
Hey, I assume this is public domain. But it would be nice to make this explicit - or if not to publish that.

**Comment by iumez** [comment]
Agreed. Having a license would clear confusion and make it clear to what extent others can use this work.
